### PR TITLE
Initial ebclfsa image and demo application

### PR DESCRIPTION
**Pull request by MofX**
# Changes

_(Provide a brief description of the changes in this pull request)_

# Dependencies:

_(Provide list of pull requests with dependecies with this pull request)_

# Tests results

```bash
<!-- add Robot test CLI logs here. -->
```

# Developer Checklist:

- [ ] Test: Changes are tested
- [ ] Doc: Documentation has been updated 
- [ ] Git: Informative git commit message(s)
- [ ] Issue: If a related GitHub issue exists, linking is done

## Checklists for documentation

- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [ ] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [ ] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [ ] Level of detail: the content matches the detail level necessary for the reviewed object

# Reviewer checklist:

- [x] Review: Changes are reviewed
- [x] Review: Tested by the reviewer

## Checklists for documentation

- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [x] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [x] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [x] Level of detail: the content matches the detail level necessary for the reviewed object

### CODE REVIEW

File: apps/ebclfsa-demo/Readme.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Depending on the host running vscode, the 'CTRL+E' key combination might be in use. If that is the case for you, open the vscode Commmand Palette (CTRL+SHIFT+P) and search for 'Preferences: Open Keyboard Shortcuts (JSON)'. In the JSON file that opens, add the following JSON to unset the Ctrl+E key binding.
+
+```json
+[
+    {
+        "key": "ctrl+e",
+        "command": "-workbench.action.quickOpenNavigateNextInEditorPicker"
+    }
+]
+```
```
Comment (MofX) [inline apps/ebclfsa-demo/Readme.md:117]: @antonjahn This shouldn't be necessary.
This PR also adds https://github.com/Elektrobit/ebcl_template/pull/57/commits/204de39f6447f12d34d91cdfb3d521e86d33cc24 which disables keybinds when the terminal has the focus

File: apps/ebclfsa-demo/Readme.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+To test executing a not allowed syscall `li_demo -p` can be executed, which calls `ioctl(0, 0)` and yields the following message:
+
+    SDK:handler_do_el0_svc_pre: syscall __NR_ioctl (29) is not allowed
+
+NOTE: The current state of the syscalls whitelist does not reflect the final list of allowed syscalls!
+
+## Interacting with the hypervisor (advanced)
+
+The hypervisor console can be accessed using the 'CTRL+E .' key combination.
```
Comment (MofX) [inline apps/ebclfsa-demo/Readme.md:104]: @antonjahn I wouldn't call this "THE" hypervisor console. What you can access here is the console muxer and I don't even know if it adds any value to be able to access it in the current state (i.e. as long as the hi vm has no way of ineracting with it using the console)

File: apps/ebclfsa-demo/Readme.md (hunk 1 above)
Comment (MofX) [reply apps/ebclfsa-demo/Readme.md]: droped with latest force-push

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
 {
-	"name": "EB corbos Linux SDK 1.4.0",
-	"image": "ghcr.io/elektrobit/ebcl_dev_container:v1.4.4",
+	"name": "EB corbos Linux SDK 1.4.2",
```
Comment (antonjahn) [inline .devcontainer/devcontainer.json:4]:
```suggestion
	"name": "EB corbos Linux SDK 1.4.5",
```

**Review by antonjahn — APPROVED** [review]
Awesome! Thanks @MofX

**Pull request merged**